src/main/java/com/wseemann/ecp/request/KeyPressRequest.java

```diff
@@ -10,6 +14,10 @@ final public class KeyPressRequest extends ECPRequest<Void> {
 	
 	public KeyPressRequest(String url, String key) {
 		super(url);		
+		try {
```

Comment:
Can you annotate this constructor as throwing UnsupportedEncodingException and remove the try/catch so the person implementing the call handle the exception? With this approach the exception is swallowed and it's probably important for the developer to know that we are attempting to encode the key they are passing.

Reply:
OK, I updated the constructors so that they let the exception through.

I also patched RoMote in this PR: https://github.com/wseemann/RoMote/pull/47